pkg/kubernetes/client.go

```diff
@@ -39,24 +38,24 @@ func NewClient(loadTestClient loadTestV1.LoadTestInterface, kubeClient kubernete
 	}
 }
 
-// CreateLoadTest creates new load test from given request data
-func (c *Client) CreateLoadTest(ctx context.Context, loadTest *apisLoadTestV1.LoadTest) (string, error) {
-	c.logger.Debug("Creating load test CR ...")
-
+// ListLoadTests lists the load test from given request data
+func (c *Client) ListLoadTests(ctx context.Context, loadTest *apisLoadTestV1.LoadTest) (*apisLoadTestV1.LoadTestList, error) {
```

Comment:
Probably name should be something like `GetLoadTestsByLabel`?
Because I assume if it is a ListLoadTests - it will give me a full list of load tests.

Reply:
Yeah, It's giving the full list of load tests, that's why I've iterated. But I can change the name, not a big deal